catalog/app/components/Pagination/index.js

```diff
@@ -0,0 +1,181 @@
+import assert from 'assert';
+import React, { PropTypes as PT } from 'react';
+import {
+  compose,
+  defaultProps,
+  pure,
+  setDisplayName,
+  setPropTypes,
+  withHandlers,
+  withProps,
+} from 'recompose';
+import styled from 'styled-components';
+import { FormattedMessage as FM } from 'react-intl';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
+import IconButton from 'material-ui/IconButton';
+import { grey500 } from 'material-ui/styles/colors';
+
+import MIcon from 'components/MIcon';
+
+import { DEFAULT_PER_PAGE, DEFAULT_PER_PAGE_OPTIONS } from './constants';
+import messages from './messages';
+
+
+const UNIT_SIZE = '24px';
+
+
+const Controls = styled.div`
+  display: flex;
+  height: ${UNIT_SIZE};
+  justify-content: space-between;
+  margin-top: 16px;
+`;
+
+const Chevron = compose(
```

Comment:
curious if `recompose` affects webpack size of JS? you can `npm run build` to see how much bigger it makes the webpack fragments.

Reply:
recompose already was in the bundle because it's MUI's dependency (that's why i pinned it to the specific (a little outdated) version).